src/flamegraph.js

```diff
@@ -437,12 +437,14 @@ export default function () {
             g.exit()
                 .remove()
 
-            g.on('mouseover', function (d) {
-                if (tooltip) tooltip.show(d, this)
-                detailsHandler(labelHandler(d))
-            }).on('mouseout', function () {
+            g.on('mouseout', function () {
                 if (tooltip) tooltip.hide()
                 detailsHandler(null)
+            }).on('mousemove', function (d) {
+                const chartBoundingClientRect = svg._groups[0][0].parentNode.getBoundingClientRect()
+                const chartRightBoundary = chartBoundingClientRect.x + chartBoundingClientRect.width
```

Comment:
This might be moved up as a separate function to fetch chart boundary

Reply:
Ok